### PR TITLE
chore: dockernize

### PR DESCRIPTION
**Pull request by bootjp**

### CODE REVIEW

**Comment by Kikobeats** [comment]
Hello, thanks for the PR.

Can you clarify what you need this as a thing living inside the project?

Even I merge this the image will be not released at the Docker Hub registry.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM node:16
```
Comment (Kikobeats) [inline Dockerfile:1]: is it a way to bind this to the latest lts version?

Comment (bootjp) [reply Dockerfile]: thank you for review, fixed it.

Comment (Kikobeats) [reply Dockerfile]: it should to point to latest LTS (right now is v14)

**Comment by bootjp** [comment]
Thank you for merging.

**Pull request merged**